internal/ovirt-mini-api.go

```diff
@@ -75,9 +77,41 @@ func NewOvirt(configReader io.Reader) (OvirtApi, error) {
 	o.Connection.Password = viper.GetString("password")
 	o.Connection.Insecure = viper.GetBool("insecure")
 	o.Connection.CAFile = viper.GetString("cafile")
+
+	username, password := loadCredentials()
+	if username != "" {
+		o.Connection.Username = username
+	}
+	if password != "" {
+		o.Connection.Password = password
+	}
+
 	return &o, nil
 }
 
+func loadCredentials() (string, string) {
+	cred, exist := os.LookupEnv("OVIRT_CONNECTION_CREDENTIAL_FILE")
+	if exist {
+		file, err := ioutil.ReadFile(cred)
+		if err != nil {
+			glog.Warningf("Failed reading connection credential file from" +
```

Comment:
Could this cause a problem? It appears that writing to stdout does not play well with flexvolume plugins.

Reply:
yes it is a problem. use the log variable for that, which is logging to syslog(look at line 41 ) and that doesn't interfere with the stdout.